report.md

```diff
@@ -54,29 +58,66 @@ The JWT will be:
 
 In addition, using HTTPS as our only mode of transfer across channels will prevent any potential leaks from HTML5 Web Storage during transfers. It also serves as a more efficient method to ensure traffic is encrypted instead of having to deploy encryption algorithms when transferring over unsecured HTTP routes.
 
+### Security for Server & Database Communication
+
 We will be protecting our system by:
-1. Using HTTPS to ensure confidentiality in data transfer between the Client & the Server.
+1. Using HTTPS to ensure confidentiality and integrity in data transfer.
 1. Disallowing executables to be uploaded into the database. 
+1. Sanitising user input. This helps to prevent SQL injections & XSS (especially if the input field is a custom type, such as JSON).
 1. Using an anti-virus scanner to scan through image and video files that will be uploaded into the database.
 
 ---
 
 ## Subsystem 2 (Interface for Therapists & Patients)
-Therapists:
-1. Able to list patients under their charge
-1. Select patients' records to view but not change
+This subsystem provides the web interface that will be used by Therapists, Patients and Administrators to access the Health Record System.
+
+### Therapists Capabilities:
+1. List all patients under their charge
+1. Select and read patients' records only
 1. Create new records
 1. Edit their own created records
 1. Print out reports
 
-Patients:
-1. Able to view but not change their medical data
+### Therapist's Interface
+After logging in, therapists would be able to list all their patients under 'My Patient' tab.
+![Therapist list all Patients](https://github.com/IFS4205-2018-Sem1-Team1/design-report/blob/subsystem-2/images/Therapist%20-%20List%20their%20patients.PNG?raw=true)
+
+The therapist can then click on the patient's name to view the specific patient's records.
+![therapist read Patient records](https://github.com/IFS4205-2018-Sem1-Team1/design-report/blob/subsystem-2/images/Therapist%20-%20Read%20Bob's%20Records.PNG?raw=true)
+
+- On this page, the therapist would have an overview of all the records of that Patient that the therapist has been given permission to view.
+- The therapist would only have read-only access to the records not owned by the therapist, and write access to records create by the therapist.
+- 'Owner' tab indicates which therapist created that record.
+- To retrieve the files to read the full details of the record, the therapist can click on the filename under the 'Document' tab.
+- The therapist can also use this interface to 'Create' new records and notes, 'Edit' them, and 'Print' these records.
+

```

Comment:
Does print mean download in a word file or something?

Reply:
Nope, I am under the impression it is just printing out whatever record the therapist is looking at into hardcopy straight. Cause when you go to a hospital, sometimes you see the nurses ferrying out a trolley full of medical dockets. I was thinking along the line of this kind of print out. Download then print out seems like double effort haha